hudi-spark-datasource/hudi-spark/src/main/scala/org/apache/spark/sql/hudi/command/CreateHoodieTableCommand.scala

```diff
@@ -198,7 +198,7 @@ object CreateHoodieTableCommand {
     val schemaJsonString = schema.json
     // Split the JSON string.
     val parts = schemaJsonString.grouped(threshold).toSeq
-    properties.put(DATASOURCE_SCHEMA_NUMPARTS, parts.size.toString)
+    properties.put(DATASOURCE_SCHEMA_PREFIX + "numParts", parts.size.toString)
```

Comment:
why need this change?

Reply:
Yea i was curious about this too.

Reply:
in spark3.2 "DATASOURCE_SCHEMA_NUMPARTS" is deleted.